frontend/src/components/layout/ContentGroup.vue

```diff
@@ -5,10 +5,15 @@ Displays the content wrapped inside a card.
 <template>
   <v-sheet class="ec-content-group mb-8 mt-2">
     <slot name="title">
-      <div v-if="title" class="ec-content-group__title py-1 subtitle-1">
+      <component
+        :is="titleTag"
+        v-if="title"
+        class="ec-content-group__title py-1 subtitle-1"
+      >
         <v-icon v-if="icon" left>{{ icon }}</v-icon>
         {{ title }}
-      </div>
+        <slot name="title-actions" />
```

Comment:
We probably want to refactor other usages of `ContentGroup` with this new functionality of title-actions. Do we want to create an issue to keep track of this?

Reply:
#3496